src/pods/about/components/members.component.module.css

```diff
@@ -34,3 +34,23 @@
   line-height: 1.4;
   margin-bottom: 0;
 }
+
+@media screen and (max-width: 780px) {
+  .member {
+    width: 60px;
+  }
```

Comment:
That 60px caused an issue with Vero's surname. Perhaps we need to modify the HTML to only limit the width of the image.